index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Microverse Hello Project</title>
+</head>
+<body>
+    <h1>Hello Microverse</h1>
+</body>
```

Comment:
- [OPTIONAL] I highly recommend you to give your `h1` element a class name and use that class name to style its content inside your CSS file. I marked it optional because it was not mentioned by previous reviewers, but it is highly recommended you work on it before moving to the next project.